src/js/components/Button/doc.js

```diff
@@ -102,6 +102,9 @@ with plain Buttons.`,
               end of the anchor.`,
       )
       .defaultValue(false),
+    secondary: PropTypes.bool.description(
```

Comment:
`kind` is missing from doc.js.

Reply:
I wasn't sure I was ready to commit fully to it. I was considering having it available for experimentation, part of this request: https://grommet.slack.com/archives/C09QQEG69/p1588591623025200?thread_ts=1588591623.025200
But, not documenting it officially until we are sure we want it. Too strange?

Reply:
> Too strange?

No, I actually agree with the experimental approach.
That just means we wouldn't want to mention the `kind` approach and functionality on the release-notes.

Reply:
completely agree